jiracli/templates.go

```diff
@@ -433,18 +441,27 @@ fields:
   assignee: {{if .fields.assignee.name}}
     emailAddress: {{ or .fields.assignee.name}}
   {{- else }}
-    emailAddress: {{.fields.assignee.emailAddress}}{{end}}{{end}}{{end}}
+    emailAddress: {{.fields.assignee.emailAddress}}
+  {{end}}{{end}}
+{{end}}
 {{- if .meta.fields.reporter}}
   reporter:
-    emailAddress: {{ if .overrides.reporter }}{{ .overrides.reporter }}{{else if .fields.reporter}}{{ .fields.reporter.emailAddress }}{{end}}{{end}}
+    emailAddress: {{ if .overrides.reporter }}{{ .overrides.reporter }}{{else if .fields.reporter}}{{ .fields.reporter.emailAddress }}{{end}}
+{{end}}
```

Comment:
Keep `{{-` or `-}}` to avoid the extra blank lines. Also, check if we can use [ternary](https://masterminds.github.io/sprig/defaults.html#ternary) here.
```suggestion
    emailAddress: {{ ternary .overrides.reporter .overrides.reporter (ternary .fields.reporter .fields.reporter.emailAddress "")  }}
{{- end}}
```

And with [coalesce](https://masterminds.github.io/sprig/defaults.html#coalesce)
```suggestion
    emailAddress: {{ coalesce .overrides.reporter (.fields.reporter | .fields.reporter.emailAddress "") }}
{{- end}}
```